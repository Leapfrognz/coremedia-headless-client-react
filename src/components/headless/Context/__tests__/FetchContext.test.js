// @flow
import React from 'react';
import renderer from 'react-test-renderer';

import FetchContext from '../FetchContext';

describe('FetchContext', () => {
  it('should render correctly with default value', () => {
    const tree = renderer
      .create(<FetchContext.Consumer>{value => value}</FetchContext.Consumer>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should render correctly with provided value', () => {
    const tree = renderer
      .create(
        <FetchContext.Provider value="TEST">
          <FetchContext.Consumer>{value => value}</FetchContext.Consumer>
        </FetchContext.Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
