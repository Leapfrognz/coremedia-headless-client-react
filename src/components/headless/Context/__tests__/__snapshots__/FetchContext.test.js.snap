// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FetchContext should render correctly with default value 1`] = `null`;

exports[`FetchContext should render correctly with provided value 1`] = `"TEST"`;
